internal/server/event_handlers.go

```diff
@@ -336,11 +338,12 @@ func createNotifier(ctx context.Context, kubeClient client.Client, provider *api
 		}
 
 		if val, ok := secret.Data["proxy"]; ok {
-			proxy = strings.TrimSpace(string(val))
-			_, err := url.Parse(proxy)
+			deprecatedProxy = strings.TrimSpace(string(val))
+			_, err := url.Parse(deprecatedProxy)
 			if err != nil {
 				return nil, "", fmt.Errorf("invalid 'proxy' in secret '%s/%s'", secret.Namespace, secret.Name)
 			}
+			log.FromContext(ctx).Error(nil, "warning: specifying proxy with 'proxy' key in secret is deprecated, use ProxySecretRef with 'address' key instead. This will be removed in v1.")
```

Comment:
I know we are logging about `spec.proxy` in the controller, but we should probably log about it while handling events as well

Reply:
Agreed!

Reply:
Actually, I think we should even remove the log we added on the controller

Reply:
[4cc3181](https://github.com/fluxcd/notification-controller/pull/1133/commits/4cc31810dc9de2f911544d8d17963a567c0eb2a5)